src/js/background/assignManager.js

```diff
@@ -198,7 +198,7 @@ window.assignManager = {
     }
 
     if (!result.proxy.mozProxyEnabled) {
-      return [{ ...result.proxy, proxyDNS: true }];
+      return { ...result.proxy, proxyDNS: true };
```

Comment:
Do we have to fix line 191, 197, and 205 too?

Reply:
Not that I could tell from the end-to-end testing.

Reply:
These brackets, in particular!..(🌪)

So after updating to an 8.0.6 with this change, proxy settings configured in 8.0.5 or earlier should just work again?

Reply:
Yes, it should just work.

Reply:
It could be. TBH I looked at [the code changes that caused the bug](https://github.com/mozilla/multi-account-containers/pull/2258/files), and I noticed that I unneccessarily changed [this line](https://github.com/mozilla/multi-account-containers/pull/2258/files#diff-b9770584df24d2a25f9eb106e0dfc4d079bb6ce210ce18c933f7f53f5657789fR201) to return an array instead of an object like it was doing before. So I turned it back into an object and it fixed the bug. 🤷 

I'll show people from the add-ons team this issue and see if they think it's a bug in Firefox or maybe the docs aren't clear?